<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductWidget</class>
 <widget class="QWidget" name="ProductWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nameProduct">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя товара</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>401</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelShop_1">
        <property name="text">
         <string>Emall.by</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShop_2">
        <property name="text">
         <string>Соседи</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShop_3">
        <property name="text">
         <string>Корона</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShop_4">
        <property name="text">
         <string>Санта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShop_5">
        <property name="text">
         <string>Евроопт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShop_6">
        <property name="text">
         <string>Гиппо</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShop_7">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelPriceFirstShop">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPriceSecondShop">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPriceThirdShop">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPriceFourthShop">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPriceFifthShop">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPriceSixthShop">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPriceSeventhShop">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="categoryProduct">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Категория товара</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
